pkgs/development/python-modules/authres/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchPypi, openssl, makeWrapper, buildPythonPackage
+, pytest, dnspython, pynacl }:
+
+buildPythonPackage rec {
+  pname = "authres";
+  version = "1.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1mcllhrwr23hwa2jn3m15k29ks1205ymwafjzchh8ma664hnzv6v";
+  };
+
+  checkInputs = [ pytest ];
```

Comment:
> Ran 0 tests in 0.000s

You might need to specify an appropriate `checkPhase`.

Reply:
@leenaars It should be
```
checkInputs = [ ];
checkPhase = ''
  # run doctests
  python -m authres
'';

Reply:
I tried, but the checks are WIP.  I cannot see how they should currently work. I suggest leaving as is.

Reply:
Well, they work if you use what I wrote above.